pkg/minikube/cluster/ip.go

```diff
@@ -72,6 +72,25 @@ func HostIP(host *host.Host) (net.IP, error) {
 		re = regexp.MustCompile(`(?s)Name:\s*` + iface + `.+IPAddress:\s*(\S+)`)
 		ip := re.FindStringSubmatch(string(ipList))[1]
 		return net.ParseIP(ip), nil
+	case driver.Parallels:
+		cmd := "prlsrvctl"
```

Comment:
cmd is used as different meaning acros the code, please rename to "bin" for better consisttancy

Reply:
Done. Renamed to `bin`